Examples/HttpAnalyzer/main.cpp

```diff
@@ -434,10 +444,18 @@ void analyzeHttpFromLiveTraffic(PcapLiveDevice* dev, bool printRatesPeriodicaly,
 	if (!dev->open())
 		EXIT_WITH_ERROR("Could not open the device");
 
-	// set a port 80 filter on the live device to capture only HTTP packets
-	PortFilter httpPortFilter(80, SRC_OR_DST);
-	if (!dev->setFilter(httpPortFilter))
-		EXIT_WITH_ERROR("Could not set up filter on device");
+	if (GlobalConfig::getInstance().setDstFilterPort)

```

Comment:
instead of add a `GlobalConfig` object, why can't we simply add the port as additional parameter to `analyzeHttpFromLiveTraffic()`?